tests/EdFi.Security.DataAccess.UnitTests/StubSecurityContext.cs

```diff
@@ -27,9 +27,11 @@ public static ISecurityContext GetMockedSecurityContext()
             securityContext.Actions = GetFakeDbSet<Action>().SetupData();
             securityContext.AuthorizationStrategies = GetFakeDbSet<AuthorizationStrategy>().SetupData();
             securityContext.ClaimSets = GetFakeDbSet<ClaimSet>().SetupData();
-            securityContext.ClaimSetResourceClaims = GetFakeDbSet<ClaimSetResourceClaim>().SetupData();
+            securityContext.ClaimSetResourceClaimActions = GetFakeDbSet<ClaimSetResourceClaimAction>().SetupData();
             securityContext.ResourceClaims = GetFakeDbSet<ResourceClaim>().SetupData();
-            securityContext.ResourceClaimAuthorizationMetadatas = GetFakeDbSet<ResourceClaimAuthorizationMetadata>().SetupData();
+            securityContext.ResourceClaimActions = GetFakeDbSet<ResourceClaimAction>().SetupData();
+            securityContext.ClaimSetResourceClaimActionAuthorizationStrategyOverrides = GetFakeDbSet<ClaimSetResourceClaimActionAuthorizationStrategyOverrides>().SetupData();
```

Comment:
@semalaiappan Are these last two "sets" on the EF DbContext actually used anywhere? I would think they would not be because the data in them should be accessed through collection properties on the entities returned by the "sets" `ResourceClaimActions` and `ClaimSetResourceClaimActions`.

Reply:
it's used here

Reply:
var resourceClaimActionAuthorizationStrategies = context.ResourceClaimActionAuthorizationStrategies.Include(rcaas => rcaas.AuthorizationStrategy)
                                    .Include(rcaas => rcaas.ResourceClaimAction)
                                    .ToList();